index.html

```diff
@@ -4,9 +4,167 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
     <title>Douglas' Portifolio page</title>
 </head>
 <body>
-    <h2>Portifolio</h2>
+<div class="main-layout">
+
+    <nav>
+        <span>Luzinda Douglas</span> 
+        <img src="https://img.icons8.com/ios-glyphs/30/ffffff/menu--v1.png" alt="Hamburger menu icon">  
```

Comment:
Kindly remove the `nav` tag from the whole section and only wrap the hamburger menu in the `nav` tag 😃

- _The `nav` tag is for semantics reasons and later on you will add a navigation menu that opens up when you click on this hamburger icon._